tests/storage/test_base.py

```diff
@@ -209,3 +209,63 @@ def test_delete_one(self) -> Generator["defer.Deferred[object]", object, None]:
         self.mock_txn.execute.assert_called_with(
             "DELETE FROM tablename WHERE keycol = ?", ["Go away"]
         )
+
+    @defer.inlineCallbacks
```

Comment:
I guess twisted/trial doesn't let you write test methods using `async def ...`? :(

Reply:
They do, but I think not on our lowest bound. I started testing it and got into some dependency hell and stopped.